controller/src/beerocks/master/tasks/association_handling_task.cpp

```diff
@@ -155,7 +155,7 @@ void association_handling_task::work()
             parent_mac); // the bssid which will be reported. for all bssid, use wildcard "ff:ff:ff:ff:ff:ff"
         //measurement_request.params.use_optional_ssid = true;
         measurement_request->params().expected_reports_count = 1;
-        //mapf::utils::copy_string(measurement_request.params.ssid, database.get_hostap_vap_ssid(parent_mac).c_str(), sizeof(measurement_request.params.ssid));
+        //string_utils::copy_string(measurement_request.params.ssid, database.get_hostap_vap_ssid(parent_mac).c_str(), sizeof(measurement_request.params.ssid));
```

Comment:
Please either remove the commented code or explain why do you keep it.

Reply:
I didn't comment out this line, simply updated the commented out call.
Maybe it's a disabled feature? Maybe the task doesn't run without this comment? In any case, it's not part of this PR.